modules/tinymce/src/plugins/lists/main/ts/core/ListNumbering.ts

```diff
@@ -83,6 +86,12 @@ const parseStartValue = (start: string): Optional<ListDetail> => {
         start: parseAlphabeticBase26(start).toString()
       });
 
+    case ListType.None:
+      return Optional.some({
+        listStyleType: Optional.none(),
+        start: '1'
```

Comment:
Should this be `''` instead of `'1'` as I still find it quite strange to be returning a number here for something that's supposed to be "None"?